### ISSUE TITLE
ffff

### ISSUE THREAD

**Issue by nmhkahn** [nmhkahn/test#1]
@pawni he's code works good, but have to change like below

`def BatchNorm(inputT, is_training=True, scope=None):
    return tf.cond(is_training,
                lambda: batch_norm(inputT, is_training=True,
                                   center=False, updates_collections=None, scope=scope),
                lambda: batch_norm(inputT, is_training=False,
                                   updates_collections=None, center=False, scope=scope, reuse = True))
`

And when run in training or test time,

`
""" when training"""
sess.run([opt, loss], feed_dict={x: bx, y: by, is_training=True})

""" when test """
sess.run([opt, loss], feed_dict={x: bx, y: by, is_training=False})
`

This code works, but like [#3265](https://github.com/tensorflow/tensorflow/issues/3265) says it will be great if `tf.contrib.layers.batch_norm` require `is_training` variable as a `tf.plcaeholer`.
